Marlin/Marlin_main.cpp

```diff
@@ -5899,12 +5899,16 @@ inline void gcode_M410() { stepper.quick_stop(); }
 
   /**
    * M421: Set a single Mesh Bed Leveling Z coordinate
+   * Eter with 'M421 Xmm Ymm Zmm' or with 'M421 In Jm Zmm'
```

Comment:
Minor misspelling. And the word "with" isn't used _quite this way_ in formal English. Better phrasing:
```cpp
* Use either 'M421 X<mm> Y<mm> Z<mm>' or 'M421 I<xindex> J<yindex> Z<mm>'
 ```
Sorry to be so pedantic. I don't mind merging this and just fixing it up later.